addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -971,4 +971,8 @@ section, .oe_img_bg, [data-oe-shape-data] {
     p, h1, h2, h3 {
         margin-bottom: 1rem;
     }
+    & > i,
+    & > span {
+        cursor: pointer;
+    }
```

Comment:
Could use the bootstrap class on the elements directly.

Reply:
Same argument applies here, more clear and robust code.

If we do the opposite and ever change the DOM structure and change this CSS accordingly, it means that old blocks will stop working as well, so there is no winners here.